includes/class-wc-admin-actionscheduler-wppoststore.php

```diff
@@ -36,23 +36,15 @@ protected function create_post_array( ActionScheduler_Action $action, DateTime $
 
 	/**
 	 * Forcefully delete all pending WC Admin scheduled actions.
-	 *
 	 * Directly trashes items from in database for performance.
+	 *
+	 * @param array $action_types Array of actions to delete.
 	 */
-	public function clear_pending_wcadmin_actions() {
+	public function clear_pending_wcadmin_actions( $action_types ) {
 		global $wpdb;
 
 		// Cancel all pending actions by trashing the posts.
 		// Action Scheduler will handle the cleanup.
-		$action_types = array(
-			WC_Admin_Reports_Sync::QUEUE_BATCH_ACTION,
-			WC_Admin_Reports_Sync::QUEUE_DEPEDENT_ACTION,
-			WC_Admin_Reports_Sync::CUSTOMERS_IMPORT_BATCH_ACTION,
-			WC_Admin_Reports_Sync::ORDERS_IMPORT_BATCH_ACTION,
-			WC_Admin_Reports_Sync::ORDERS_IMPORT_BATCH_INIT,
-			WC_Admin_Reports_Sync::SINGLE_ORDER_IMPORT_ACTION,
-		);
-
 		foreach ( $action_types as $action_type ) {
```

Comment:
Since this is a public function `$action_types` should probably be cast to an array.